report-service/src/main/resources/web/template/report.hbs

```diff
@@ -0,0 +1,146 @@
+                    <h4 id="{{testType.anchor}}">{{message}}</h4>
+                </div>
+        {{/if}}
+    {{/with}}
+    {{#with data.uniqueErrorsTestResults}}
+        {{#if passed}}
+            <div class="d-flex align-center mt-4 mb-2">
+                {{svg "web/img/embedded-success.svg"}}
+                <h4 id="{{testType.anchor}}">{{message}}</h4>
+            </div>
+            <p class="ml-2 muted">Nothing to report</p>
+        {{else}}
+            <div class="d-flex align-center mt-4 mb-2">
+                {{svg "web/img/embedded-danger.svg"}}
+                <h4 id="{{testType.anchor}}">{{message}}</h4>
+            </div>
+        {{/if}}
+    {{/with}}
+    {{#if showTopEvents}}
+    <table class="table table-striped">
+        <thead>
+        <tr>
+            <th>Event</th>
+            <th>Application(s)</th>
+            <th>Introduced by</th>
+            <th>Volume</th>
+        </tr>
+        </thead>
+        <tbody>
+        {{#each data.topEvents}}
+            <tr>
+                <td>
+                    <a href="{{this.arcLink}}" target="_blank">{{eventSummary}}</a>
+                </td>
+                <td>{{this.applications}}</td>
+                <td>{{this.introducedBy}}</td>
+                <td>{{this.hits}}</td>
+            </tr>
+        {{/each}}
+        </tbody>
+    </table>
+    {{/if}}
+
+    {{#with data.criticalErrorsTestResults}}
+        {{> reportSection data.criticalErrorsTestResults}}
+    {{/with}}
+
+    {{#with data.regressionErrorsTestResults}}
+        {{#if passed}}
+            <div class="d-flex align-center mt-4">
+                {{svg "web/img/embedded-success.svg"}}
+                <h4 id="{{testType.anchor}}">{{message}}</h4>
+            </div>
+            <p class="ml-2 muted">Nothing to report</p>
+        {{else}}
+            <div class="d-flex align-center mt-4 mb-2">
+                {{svg "web/img/embedded-danger.svg"}}
+                <h4 id="{{testType.anchor}}">{{message}}</h4>
+            </div>
+            <table class="table table-striped">
+                <thead>
+                <tr>
+                    <th>Event</th>
+                    <th>Application(s)</th>
+                    <th>Introduced by</th>
+                    <th>Volume/Rate</th>
+                    <th>Type</th>
+                </tr>
+                </thead>
+                <tbody>
+                {{#each events}}
+                    <tr>
+                        <td>
+                            <a href="{{this.arcLink}}" target="_blank">{{eventSummary}}</a>
+                        </td>
+                        <td>{{this.applications}}</td>
+                        <td>{{this.introducedBy}}</td>
+                        <td>{{this.hits}}</td>
+                        <td>{{this.type}}</td>
+                    </tr>
+                {{/each}}
+                </tbody>
+            </table>
+        {{/if}}
+    {{/with}}
```

Comment:
I think the regression section can be removed.

Reply:
keep it based on whether this changes in the future. Can be removed by not adding the query parameters for it.